.github/workflows/maven.yml

```diff
@@ -29,9 +29,9 @@ jobs:
       - name: Set up JDK 17
         uses: actions/setup-java@v2
         with:
-          distribution: 'zulu'
+          distribution: 'adopt-openj9'
```

Comment:
why OpenJ9 and not `temurin` which we nowadays use for our docker images?

Reply:
openj9 is not working. will switch to temurin.